.github/workflows/continuous-integration.yml

```diff
@@ -336,10 +336,73 @@ jobs:
         if: ${{ always() }}
         run: node ./script/github-actions/stylelint-annotation-format.js
 
+  cypress-tests-prep:
```

Comment:
Is this only on master or on all branches as well? I ask cause the files change output `on_push` differs from `pull_request` or when merged to master

Reply:
Marking as resolved as I reviewed in detail